Podcast/SeriesDetailHeaderView.swift

```diff
@@ -178,13 +193,15 @@ class SeriesDetailHeaderView: UIView, UICollectionViewDelegate, UICollectionView
             make.leading.trailing.equalToSuperview()
             make.height.equalTo(tagsViewHeight)
             make.top.equalTo(viewSeparator.snp.bottom).offset(tagsViewTopOffset)
+            make.bottom.equalToSuperview().inset(tagsViewTopOffset)
         }
         
         episodeSeparator.snp.makeConstraints { make in
-            make.centerX.equalToSuperview()
             make.height.equalTo(episodeSeparatorHeight)
-            make.width.equalToSuperview()
-            make.top.equalTo(tagsCollectionView.snp.bottom).offset(tagsViewTopOffset)
+            make.top.equalTo(contentContainer.snp.bottom)
+            make.leading.equalToSuperview()
```

Comment:
`make.leading.trailing.bottom.equalToSuperview()` instead of 3 lines

Reply:
holy...